incubator/kube-aws-ingress-controller/templates/daemonset.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1beta2
+kind: DaemonSet
+metadata:
+  name: {{ template "kube-aws-ingress-controller.fullname" . }}
+  namespace: kube-system
+  labels:
+    app: {{ template "kube-aws-ingress-controller.name" . }}
+    chart: {{ template "kube-aws-ingress-controller.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "kube-aws-ingress-controller.name" . }}
+      release: {{ .Release.Name }}
+      component: ingress
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      name: {{ template "kube-aws-ingress-controller.fullname" . }}
+      labels:
+        app: {{ template "kube-aws-ingress-controller.name" . }}
+        release: {{ .Release.Name }}
+        component: ingress
+    spec:
+      serviceAccountName: {{ template "kube-aws-ingress-controller.serviceAccountName" . }}
+      hostNetwork: true
+      containers:
+      - name: {{ template "kube-aws-ingress-controller.name" . }}
+        image: "{{ .Values.daemonset.image.repository }}:{{ .Values.daemonset.image.tag }}"
+        imagePullPolicy: "{{ .Values.daemonset.image.pullPolicy }}"
+        ports:
+        - name: ingress-port
+          containerPort: 9999
+          hostPort: 9999
+        args:
+          - "skipper"
+          - "-kubernetes"
+          - "-kubernetes-in-cluster"
+          - "-address=:9999"
+          - "-proxy-preserve-host"
+          - "-serve-host-metrics"
+          - "-enable-ratelimits"
+          - "-experimental-upgrade"
+          - "-metrics-exp-decay-sample"
+          - "-kubernetes-https-redirect=true"
```

Comment:
I would add these to enable monitoring by default:

	      - "-lb-healthcheck-interval=3s"
	      - "-metrics-flavour=codahale,prometheus"
	      - "-enable-connection-metrics"